doc/source/ray-core/examples/map_reduce.ipynb

```diff
@@ -120,7 +120,7 @@
    "id": "52009879",
    "metadata": {},
    "source": [
-    "We use the `apply_map` function on a large collection of documents by marking it as a task in Ray using the `@ray.remote` decorator.\n",
+    "We use the `apply_map` function on a large collection of documents by marking it as a task in Ray using the [`@ray.remote`](https://docs.ray.io/en/latest/ray-core/api/doc/ray.remote.html) decorator.\n",
```

Comment:
In notebook, is there a way to use things like
```
:func:`ray.remote <ray.remote>`
```
instead of the absolute url path?